### PR TITLE
feat(user): adding role_ids to user_create spec [TA-3389]

### PR DESCRIPTION
**Pull request by galbino**
# Description

Added `role_ids` array to user create spec

Re TA-3389

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own changes
- [X] I have run `yarn test` to make sure my changes pass all linters
- [X] I have pulled the latest changes from the upstream `main` branch

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: reference/request-bodies/user_create_request.yaml [hunk 1]
```diff
@@ -26,3 +26,15 @@ properties:
...
     maxLength: 200
+
+  role_ids:
+    type: array
+    minItems: 1
+    description:
+      A list of role ids that will be assigned to the user being created.
+      The creator must have `roles.write` or the role that is being assigned.
+    items:
+      type: string
```
Comment (epigos) [inline reference/request-bodies/user_create_request.yaml:37]:
```suggestion
    items:
      type: string
      format: uuid
```

**Review by epigos — COMMENTED** [review]
I know this is not part of the ticket but the schema `display_name` in the API specs for user create has `display_name` instead of `name` - https://github.com/gr4vy/gr4vy-openapi/blob/8d97617e9b32f9fa6c12f51d09c193ddd1d86440/reference/request-bodies/user_create_request.yaml#L10. Can we make the changes as part of this PR?

File: reference/request-bodies/user_create_request.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ x-tags:
   - Users
 
 properties:
-  display_name:
+  name:
     type: string
     description:
       An external identifier that can be used to match the buyer against your
```
Comment (epigos) [inline reference/request-bodies/user_create_request.yaml:13]: Do we need to update the descriptions to reflect the user's full name instead of the buyer?

Comment (galbino) [reply reference/request-bodies/user_create_request.yaml:13]: wow, that's my bad, I think I didn't notice that

**Review by epigos — APPROVED** [review]

**Pull request merged**